README.md

```diff
@@ -162,15 +147,6 @@ I would like to thank the entire Microverse staff and colleagues for making this
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ <a name="faq"></a>
-
-N/A
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
```

Comment:
- Kindly, your `mit` link is broken because you didn't define the `LICENSE.md` file it is [MIT.md](https://github.com/Mylo16/db-query/blob/update-animals/MIT.md) so please update it to have professional readme as suggested by previous reviwer👍 
<img width="584" alt="Screenshot 2023-06-14 220407" src="https://github.com/Mylo16/db-query/assets/52010789/081c6200-4e62-47d2-b0a3-47805a57b0c7">